ridgeback_baxter_description/urdf/ridgeback_baxter.urdf.xacro

```diff
@@ -1,15 +1,56 @@
 <?xml version="1.0"?>
 <robot name="ridgeback_baxter" xmlns:xacro="http://www.ros.org/wiki/xacro">
-  <xacro:property name="pedestal_height" value="0.565" />
+  <xacro:arg name="gazebo" default="false"/>
+  <xacro:arg name="pedestal" default="false"/>
+  <xacro:arg name="with_baxter" default="true"/>
+
+  <xacro:property name="pedestal_height" value="-0.92" />
   <xacro:property name="pedestal_diameter" value="0.203" />
-  <xacro:property name="pedestal_offset_x" value="0.2" />
+  <xacro:property name="pedestal_offset_x" value="-0.2" />
 
+  <!-- Ridgeback URDF -->
   <xacro:include filename="$(find ridgeback_description)/urdf/ridgeback.urdf.xacro" />
-  <xacro:include filename="$(find baxter_description)/urdf/baxter.urdf" />
 
-  <joint name="ridgeback_base_to_baxter" type="fixed">
+  <xacro:if value="$(arg with_baxter)">
+    <!-- Baxter URDF -->
+    <xacro:include filename="$(find baxter_description)/urdf/baxter.urdf.xacro">
+      <xacro:arg name="gazebo" value="${gazebo}"/>
+      <xacro:arg name="pedestal" value="${pedestal}"/>
+    </xacro:include>
+  </xacro:if>
+
+  <!-- Ridgeback-Baxter Joint URDF -->
+  <joint name="pedestal_fixed" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0 0" />
+    <parent link="torso" />
+    <child link="pedestal_stand" />
+  </joint>
+  <link name="pedestal_stand">
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://ridgeback_baxter_description/meshes/pedestal_stand.dae"/>
+      </geometry>
+      <material name="darkgray">
+        <color rgba=".2 .2 .2 1"/>
+      </material>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://ridgeback_baxter_description/meshes/pedestal_stand_collision.dae"/>
+      </geometry>
+    </collision>
+    <inertial>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <mass value="30.0"/>
```

Comment:
I am actually not sure if this is the right mass or inertia for the pedestal stand...